Enforce Hub when deploying notebooks from `LIVEBOOK_APPS_PATH` (#2039)
livebook-dev/livebook: README.md

  * LIVEBOOK_SHUTDOWN_ENABLED - controls if a shutdown button should be shown
    in the homepage. Set it to "true" to enable it.

  * LIVEBOOK_TEAMS_KEY - sets the secret Livebook Teams key for creating an offline hub.

>     Must be set aside of LIVEBOOK_TEAMS_NAME and LIVEBOOK_TEAMS_OFFLINE_KEY.
> 
>   * LIVEBOOK_TEAMS_NAME - sets the Livebook Teams name for creating an offline hub.
>     Must be set aside of LIVEBOOK_TEAMS_KEY and LIVEBOOK_TEAMS_OFFLINE_KEY.
> 
>   * LIVEBOOK_TEAMS_OFFLINE_KEY - sets the Livebook Teams public key for creating an offline hub.
>     Must be set aside of LIVEBOOK_TEAMS_NAME and LIVEBOOK_TEAMS_KEY.

Reviewer: ```suggestion
    Must be set together with LIVEBOOK_TEAMS_NAME and LIVEBOOK_TEAMS_OFFLINE_KEY.

  * LIVEBOOK_TEAMS_NAME - sets the Livebook Teams name for creating an offline hub.
    Must be set together with LIVEBOOK_TEAMS_KEY and LIVEBOOK_TEAMS_OFFLINE_KEY.

  * LIVEBOOK_TEAMS_OFFLINE_KEY - sets the Livebook Teams public key for creating an offline hub.
    Must be set together with LIVEBOOK_TEAMS_NAME and LIVEBOOK_TEAMS_KEY.
```

Or "alongside".
